pkg/controllers/csi/provisioner/controller_test.go

```diff
@@ -373,7 +373,7 @@ func TestOneAgentProvisioner_Reconcile(t *testing.T) { //nolint:revive
 				},
 			},
 		}
-		installerMock := &installer.Mock{}
+		installerMock := &mocks.Installer{}
```

Comment:
could  `mocks.NewInstaller()` be used instead of `struct{}` if any expectations are defined? The lib makes sure they are correct (calling `AssertExpectations()`)

Reply:
Good point! Fixed in 1d49bd1f
Actually it showed we had some unused code